libraries/griptape_nodes_library/griptape_nodes_library/execution/switch.py

```diff
@@ -0,0 +1,216 @@
+from typing import Any
+
+from griptape_nodes.exe_types.core_types import (
+    Parameter,
+    ParameterList,
+    ParameterMode,
+    ParameterTypeBuiltin,
+)
+from griptape_nodes.exe_types.node_types import BaseNode, ControlNode
+from griptape_nodes.retained_mode.events.connection_events import DeleteConnectionRequest
+from griptape_nodes.retained_mode.events.parameter_events import GetConnectionsForParameterRequest
+from griptape_nodes.retained_mode.griptape_nodes import GriptapeNodes
+
+
+class Switch(ControlNode):
+    """A control flow node that routes execution based on matching case values.
+
+    Compares a test value against a list of cases and directs control flow
+    to the matching case's output. If no match is found, routes to the
+    "No Match" control output.
+    """
+
+    def __init__(self, name: str, metadata: dict[Any, Any] | None = None) -> None:
+        super().__init__(name, metadata, output_control_name="No Match")
+
```

Comment:
Can we add a ParameterMessage here explaining what this node is for to the user? Without an understanding of what it's for, the user will be very confused.